honeybee/typing.py

```diff
@@ -51,7 +51,8 @@ def valid_ep_string(value, input_name=''):
     and ensuring the name is not longer than 100 characters.
     """
     try:
-        val = re.sub(r'[^.\sA-Za-z0-9_-]', '', value)
+        val = ''.join(i for i in value if ord(i) < 128)  # strip out non-ascii
```

Comment:
This will remove white space in string. Is this what you want to happen for EP names?

Reply:
I'm not sure why you say that because white spaces are definitely present in the test that I wrote for this method:
https://github.com/ladybug-tools/honeybee-core/pull/10/files#diff-5e17cef320555cc0b9c2ad939932b42dR9
Is a white space not considered an ASCII character in some cases?

Reply:
Hmm. You are right. I should have run the wrong code. I should say that I still like the regular expression version. It is more explicit than ord(i) < 128. But it does work so 👍.